packages/components/psammead-radio-schedule/README.md

```diff
@@ -20,6 +20,54 @@ npm install @bbc/psammead-radio-schedule --save
 
 ## Components (WIP)
 
+## RadioSchedule
+
+### Props
+
+<!-- prettier-ignore -->
+| Argument | Type | Required | Default | Example |
+| -------- | ---- | -------- | ------- | ------- |
+| locale | string | yes | N/A | `'en-gb'` |
+| schedules | array | yes | N/A | `[{ id: '1', state: 'live', stateLabel: 'Live', startTime: '1566914061212', locale: 'en-gb', timezone: 'Europe/London', link: 'www.bbc.co.uk', brandTitle: 'This is a brand title', episodeTitle: 'This is an episode title', summary: 'This is a summary', duration: '45:00', durationLabel: 'Duration'}]` |
```

Comment:
I'm not sure about passing in all of this data as one array instead of having separate props. Should we be mapping over this data and creating the child components outside of this wrapper, and then compose them in?

Reply:
@tochwill composing components outside the wrapper will potentially bring issues with aligning the card to keep their heights consistent.